### PR TITLE
fix: give indexer some time to catch up before failing on restart

### PR DESCRIPTION
**Pull request by ppca**

### CODE REVIEW

File: chain-signatures/node/src/indexer.rs [hunk 1]
```diff
@@ -324,7 +328,7 @@ pub fn run(
                 anyhow::Ok(lake)
             }) else {
                 tracing::error!(?options, "indexer failed to build");
-                backoff(i);
+                backoff(i, 1, 120);
```
Comment (ailisp) [inline chain-signatures/node/src/indexer.rs:331]: Seems i will never decrease, and eventually i will become `max`  if indexer keep working. Do we need to restore it to a small number on a not failing case?

File: chain-signatures/contract/src/lib.rs [hunk 2]
```diff
@@ -584,6 +584,7 @@ impl VersionedMpcContract {
 #[near_bindgen]
 impl VersionedMpcContract {
     #[handle_result]
+    #[init(ignore_state)]
```
Comment (volovyks) [inline chain-signatures/contract/src/lib.rs:587]: Why do we need this change? Not sure if it's safe.

Comment (ppca) [reply chain-signatures/contract/src/lib.rs:587]: ah sorry. shouldn't have pushed this.

File: chain-signatures/node/src/indexer.rs [hunk 3]
```diff
@@ -349,6 +353,10 @@ pub fn run(
                 rt.spawn(async move { lake.run_with_context_async(handle_block, &context).await })
             };
             let outcome = rt.block_on(async {
+                if i > 0 {
+                    // give it some time to catch up
+                    backoff(i, 10, 300);
+                }
```
Comment (ChaoticTempest) [inline chain-signatures/node/src/indexer.rs:359]: we could've just had a flat `sleep(5mins)` here instead of modifying `backoff`

**Review by ChaoticTempest — APPROVED** [review]

**Pull request merged**